src/apis/storage_partitioning.js

```diff
@@ -248,14 +254,31 @@ export const gatherRequests = ({ partitioning }) => {
     ).then(res => res[0]);
 };
 
-export const applyStorage = async ({ luks, onFail, onSuccess, partitioning }) => {
+export const applyStorage = async ({ devices, luks, onFail, onSuccess, partitioning }) => {
     if (luks?.encrypted !== undefined) {
         await partitioningSetEncrypt({ encrypt: luks.encrypted, partitioning });
     }
     if (luks?.passphrase) {
         await partitioningSetPassphrase({ partitioning, passphrase: luks.passphrase });
     }
 
+    const method = await getPartitioningMethod({ partitioning });
+    if (method === "MANUAL") {
+        const requests = await gatherRequests({ partitioning });
+        const bootloaderDevice = requests.find(request => ["biosboot", "efi"].includes(request["format-type"].v))?.["device-spec"].v;
```

Comment:
I wonder if we should use [bootloaderTypes](https://github.com/rhinstaller/anaconda-webui/blob/06271e44e0fda0cfdaa2c1c6be36f75eff259611/src/helpers/storage.js#L176) here but don't insist - I guess it is not the only place we work just with these two types.

Reply:
Good point. I wiill.